src/scss/headers.scss

```diff
@@ -0,0 +1,147 @@
+.navbar {
+    padding-left: 0;
+    padding-right: 0;
+    @include themify($themes) {
+        background-color: themed('navBackground') !important;
+    }
+
+    .dropdown-menu {
+        max-width: 300px;
+        min-width: 200px;
+
+        .dropdown-item-text {
+            line-height: 1.3;
+            @include themify($themes) {
+                color: themed('darkTextColor');
+            }
+
+            span, small {
+                display: block;
+                overflow: hidden;
+                text-overflow: ellipsis;
+                white-space: nowrap;
+                &.text-muted {
+                    @include themify($themes) {
+                        color: themed('dropdownTextMuted') !important;
+                    }
+                }
+            }
+        }
+    }
+    .nav-item {
+        > .nav-link {
+            @include themify($themes) {
+                font-weight: themed('navWeight');
+            }
+        }
+        &.active > .nav-link {
+            @include themify($themes) {
+                font-weight: themed('navActiveWeight');
+            }
+        }
+    }
+}
+
+.navbar-brand {
+    margin-bottom: -20px;
+    margin-top: -20px;
+}
+
+.page-header, .secondary-header {
```

Comment:
Most of this file relates to the navBar, which is not really a 'header' in the same way `.page-header` and `.secondary-header` is. I think we just need to redistribute this file among the other files.
* move this block (`.page-header...`) and the next block (`.secondary-header...`) into `base.scss`
* rename this file (with the styles that are left in it) `misc.scss`